handbook/engineering/cloud/index.md

```diff
@@ -0,0 +1,33 @@
+# Cloud team
+
+The cloud team owns all work that is necessary to build, secure, scale, and operate our multi-tenant hosted version of Sourcegraph for customers that do not want to deploy Sourcegraph on-premise.
```

Comment:
I read the PR description, but this still feels like the "cloud team" is not responsible for on-premise version, and sounds like we now have two dedicated products, Sourcegraph Cloud and Sourcegraph Enterprise/On-premise, which is not true (as of now?).

Reply:
The mission of this team is to bring our current product to the cloud, not create a different product. 90+% of the work this team does will be relevant to all customers, regardless of how they deploy.

In an alternate world where we were cloud first and wanted to build an on-prem solution, this team would already exist and we would probably create a team similar to what the distribution team is today (and they would own "how do we deploy this product that we already have on-premise at customers).

Does this make sense?

Reply:
Gotcha, thanks!